src/OpenSpaceToolkit/Astrodynamics/Trajectory/Segment.cpp

```diff
@@ -693,10 +966,11 @@ Segment Segment::ConstantLocalOrbitalFrameDirectionManeuver(
     const Array<Shared<Dynamics>>& aDynamicsArray,
     const NumericalSolver& aNumericalSolver,
     const Shared<const LocalOrbitalFrameFactory>& aLocalOrbitalFrameFactory,
-    const Angle& aMaximumAllowedAngularOffset
+    const Angle& aMaximumAllowedAngularOffset,
+    const ManeuverConstraints& aManeuverConstraints
 )
 {
-    return {
+    return Segment {
```

Comment:
I'm not sure this works?
```suggestion
    return {
```